test/testutil/testutil_init.c

```diff
@@ -36,16 +36,15 @@ static size_t internal_trace_cb(const char *buf, size_t cnt,
         BIO_snprintf(buffer, sizeof(buffer), "TRACE[%s]:%s: ",
                      hex, OSSL_trace_get_category_name(category));
         OPENSSL_free(hex);
-        BIO_ctrl(trace_data->bio, PREFIX_CTRL_SET_PREFIX,
-                 strlen(buffer), buffer);
+        BIO_set_prefix(trace_data->bio, buffer);
```

Comment:
This call does a strdup internally which may fail and return 0 (not likely).. Does the error code need to be checked?

Reply:
Hmmm... this is low level tracing code. Failing is not an option imho and placing some error message between the trace messages might be confusing. Anyway,  If the strdup() fails, all that happens is that the prefix will be missing. So I'd tend to keep the code simple and don't do an error check here.